cmd/profile.go

```diff
@@ -108,10 +109,28 @@ Sets the peer avatar from an image path (JPEG, PNG, or GIF).`
 
 func (x *setAvatarCmd) Execute(args []string) error {
```

Comment:
The command-line client now leverages the add files method using the account thread, and then hits the profile/avatar endpoint which just announces the change to all threads.